packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +64,62 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+const props = defineProps({
+  layoutId: {
+    type: String,
+    default: null,
+  },
+})
+
+type PaneEvent = {
+  max: number
+  min: number
+  size: number
+}
+
+const PANE_SIDEBAR_SIZE = ref(25)
+const PANE_MAIN_SIZE = ref(75)
+const PANE_MAIN_TOP_SIZE = ref(45)
+const PANE_MAIN_BOTTOM_SIZE = ref(65)
+const PANE_STORAGE_KEY = `${props.layoutId}-app-pane-layout`
+
+if (!COLUMN_LAYOUT) {
+  PANE_MAIN_TOP_SIZE.value = 50
+  PANE_MAIN_BOTTOM_SIZE.value = 50
+}
+
+function setVerticalPaneEvent(event: PaneEvent[]) {
+  if (!props.layoutId) return
+
+  setLocalConfig(`${PANE_STORAGE_KEY}_vertical`, JSON.stringify(event))
+}
```

Comment:
`setVerticalPaneEvent` and `setHorizontalPaneEvent` are doing the same thing. just storing value to storage. Instead of creating two function we can go with one function. You can name that `storePaneEvent`. It will have two parameters `event` and `type`. For horizontal type will be `hr` and for vertical it will be `vr`. Then pass the argument from template itself.